### DISCUSSION TITLE
Gentoo ebuild

### DISCUSSION THREAD

**Discussion by dragetd** [mullvad/mullvadvpn-app#8903] [Feature requests & Ideas]
I know the typical Gentoo user would rather stick to wg-quick, but I wonder if anyone else is interested in an eBuild for Gentoo?

**Comment by langest** [comment]
I came here to ask more or less the same question.

**Comment by jeandestouches** [comment]
I'd be very interested by an ebuild. Any news ? Thanks

**Comment by DeadNumbers** [comment]
up

**Comment by thanosapollo** [comment]
`mullvadvpn-app` has been added to GURU

**Comment by dragetd** [comment]
It was added by @stkw0 , cool, thank you!

For anyone interested and familiar with ebuild writing, maybe we can get some crowd intelligence for feedback and improvement: https://github.com/gentoo/guru/blob/master/net-vpn/mullvadvpn-app/mullvadvpn-app-2023.5.ebuild

I will give it a spin later

**Comment by stkw0** [comment]
It is a very trivial ebuild though. I think that the main improvement that can be made is adding a service file for OpenRC.
